### DISCUSSION TITLE
SX1262 (Ra-01SH) Compatibility

### DISCUSSION THREAD

**Discussion by GrimmsterZed** [akshayabali/pico-lora#9] [General]
First off a huge thank you for making this library, from what I've beenn looking at it's really handy to use.

Unfortunately because I'm in Canada I need to use a LoRa module operating in the 915MHz band, so I am using the SX1262 chip on the Ra-01SH.

Would you be able to help with this compat, or point me in the right direction as to how I can modify the library? I was hoping the chips were similar enough to be compatible out of the box but I am not having any luck running the test scripts.

Thank you in advance

**Comment by akshayabali** [comment]
Hi @GrimmsterZed,

I once used the Ra-01S module which is the same module but operates on 413Mhz so I believe you can use the library without any modifications with the Ra-01 chip as the underlying chip and protocol is SX1262. 

I wrote the library as a side project so there might be some issues when you compile or try to run it, but you can just post any issue, and someone or I will help you with them.

**Reply by GrimmsterZed** [reply]
Thank you for reaching out!

I managed to get it to run, just had to ignore the version check. Unfortunately the 126x datasheet is a lot less detailed than the 127x datasheet, but your macros appear to be working correctly.

Unfortunately I am experiencing an issue with the Rx code. #10
